_docs/_latest/use/cli/scanner.md

```diff
@@ -37,9 +37,15 @@ forseti scanner run
 ```
 
 The external project access scanner is not part of the regular running scanners, as it may take hours to run. 
-Therefore, we now make this scanner accessible manually through the CLI:
+Therefore, we now make this scanner accessible manually through the CLI. <br />
+**Note:** You will need to create an inventory and model if it is not already
+created. 
 ```bash
+MODEL_ID=$(/bin/date -u +%Y%m%dT%H%M%S)
+forseti inventory create --import_as ${MODEL_ID}
+forseti model use ${MODEL_ID}
 forseti scanner run --scanner external_project_access_scanner
+forseti notifier run --scanner_index_id <SCANNER_INDEX_ID>
```

Comment:
Can you also mention how to get the scanner index it? I think it's part of the initial output from running forseti scanner run.